Aplib.Core/DataStructures/CircularArray.cs

```diff
@@ -1,4 +1,4 @@
-namespace Aplib.Core
+namespace Aplib.Core.DataStructures
```

Comment:
I am not a fan of `DataStructures` as this is a bit vague. Why don't we mimic `System` and call it `Aplib.Core.Collections`?

```suggestion
namespace Aplib.Core.Collections
```

Reply:
Agreed!